src/sagemaker/utils.py

```diff
@@ -1462,6 +1462,7 @@ def volume_size_supported(instance_type: str) -> bool:
             and not family.startswith("g5")
             and not family.startswith("g6")
             and not family.startswith("p5")
+            and not family.startswith("trn1")
```

Comment:
```
```
unsupported_families = ["g5", "g6", "p5", "trn1"]
return not any(family.startswith(prefix) for prefix in unsupported_families)
```
not a blocker, but I think it might be more concise to write it as above.

Reply:
thanks, i agree that that's def a better organization